deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -90,6 +86,13 @@ protected void generate(final Config config, final Path openApiFilePath, final P
         config.getOptionalValue(getCustomRegisterProvidersFormat(openApiFilePath), String.class)
                 .ifPresent(generator::withCustomRegisterProviders);
 
+        SmallRyeConfig smallRyeConfig = config.unwrap(SmallRyeConfig.class);
+        smallRyeConfig.getOptionalValues(getTypeMappingsPropertyName(openApiFilePath), String.class, String.class)
+                .ifPresent(generator::withTypeMappings);
+
+        smallRyeConfig.getOptionalValues(getImportMappingsPropertyName(openApiFilePath), String.class, String.class)
+                .ifPresent(generator::withImportMappings);
+
```

Comment:
I think we need to start thinking about an abstraction in another class to return/create all our configurations and access it here instead.